docs/user/rocket/generic_surface.rst

```diff
@@ -0,0 +1,422 @@
+- :math:`C_D` as the drag coefficient.
+
+And the forces from the coefficients are defined as:
+
+.. math::
+   \begin{bmatrix}X_A\\Y_A\\Z_A\end{bmatrix}_B =\mathbf{M}_{BA}\cdot\overline{q}\cdot A_{ref}\cdot\begin{bmatrix}C_Q\\-C_L\\-C_D\end{bmatrix}_A 
+
+Where:
+
+- :math:`\bar{q}` is the dynamic pressure.
+- :math:`A_{ref}` is the reference area used to calculate the coefficients.
+
+The moment coefficients can be defined as:
+
+- :math:`C_l` as the rolling moment coefficient.
+- :math:`C_m` as the pitching moment coefficient.
+- :math:`C_n` as the yawing moment coefficient.
+
+And the moments from the coefficients are defined as:
+
+.. math::
+   \vec{\mathbf{M}}_B=\begin{bmatrix}M_{x_A}\\M_{y_A}\\M_{z_A}\end{bmatrix}_B =\overline{q}\cdot A_{ref}\cdot L_{ref}\cdot\begin{bmatrix}C_m\\C_n\\C_l\end{bmatrix}
+
+Where:
+
+- :math:`L_{ref}` is the reference length used to calculate the coefficients.
+
+
+Aerodynamic angles
+~~~~~~~~~~~~~~~~~~
+
+There aerodynamic angles are defined in two different ways in RocketPy:
+
+- As the angle of attack (:math:`\alpha`) and the side slip 
+  angle (:math:`\beta`), which are defined in the image above. These are used
+  in the calculation of the generic surface forces and moments.
+- As the total angle of attack (:math:`\alpha_{\text{tot}}`), defined as the
+  angle between the total velocity vector and the rocket's centerline. This is
+  used in the calculation of the standard aerodynamic surface forces and moments.
+
+The partial angles are calculated as:
+
+.. math::
+   \begin{aligned}
+      \alpha &= \arctan\left(\frac{V_y}{V_z}\right) \\
+      \beta &= \arctan\left(\frac{V_x}{V_z}\right)
+   \end{aligned}
+
+The total angle of attack is calculated as:
+
+.. math::
+   \alpha_{\text{tot}} = \arccos\left(\frac{\mathbf{\vec{V}}\cdot\mathbf{z_B}}{||\mathbf{\vec{V}}||\cdot||\mathbf{z_B}||}\right)
+
+.. note::
+   When the simulation is done, the total angle of attack is accessed through
+   the :attr:`rocketpy.Flight.angle_of_attack` attribute.
+   The partial angles of attack and side slip are accessed through the
+   :attr:`rocketpy.Flight.partial_angle_of_attack` and 
+   :attr:`rocketpy.Flight.angle_of_sideslip` attributes, respectively.
+
+.. _genericsurface:
+
+Generic Surface Class
+---------------------
+
+The :class:`rocketpy.GenericSurface` class is used to define a aerodynamic
+surface based on force and moment coefficients. A generic surface is defined
+as follows:
+
+.. seealso::
+   For more information on class initialization, see 
+   :class:`rocketpy.GeericSurface.__init__` 
+
+
+.. code-block:: python
+
+   from rocketpy import GenericSurface
+   generic_surface = GenericSurface(
+      reference_area=np.pi * 0.0635**2,
+      reference_length=2 * 0.0635,
+      coeffcients={
+         "cL": "cL.csv",
+         "cQ": "cQ.csv",
+         "cD": "cD.csv",
+         "cm": "cm.csv",
+         "cn": "cn.csv",
+         "cl": "cl.csv",
+      },
+      name="Generic Surface",
+   )
```

Comment:
```suggestion
   from rocketpy import GenericSurface


   generic_surface = GenericSurface(
      reference_area=np.pi * 0.0635**2,
      reference_length=2 * 0.0635,
      coeffcients={
         "cL": "cL.csv",
         "cQ": "cQ.csv",
         "cD": "cD.csv",
         "cm": "cm.csv",
         "cn": "cn.csv",
         "cl": "cl.csv",
      },
      name="Generic Surface",
   )
```

Reply:
> Seems like the GenericSurface was never exposed to the rocketpy module

Are you sure? I believe the __init__.py files are all correct

Reply:
I was able to import like `from rocketpy import GenericSurface` from test notebooks.

Reply:
I think I have forgotten to git checkout the branch, sorry.

But the code block can still be improved